wrappers/java/build.gradle

```diff
@@ -1,11 +1,57 @@
 plugins {
     id 'java'
+    id 'application'
 }
 
 version '1.0-SNAPSHOT'
 
 sourceCompatibility = 1.8
 
+mainClassName = ''
```

Comment:
This will create extra script during the build. I created this commit to demonstrate more optimal solution that does not create additional redundant script: https://github.com/ConsenSys/EthereumRegistrationAuthority/commit/fd094851f82036bf4a74e60dcaade331f6a69916